libraries/ESP32/examples/ArduinoStackSize/ArduinoStackSize.ino

```diff
@@ -0,0 +1,37 @@
+/*
+    ESP32 Arduino creates a task to run setup() and then to execute loop() continuously
+    This task can be found at https://github.com/espressif/arduino-esp32/blob/master/cores/esp32/main.cpp
+
+    By default "loopTask" will be created with a stack size of 8KB.
+    This should be plenty for most general sketches.
+
+    There is a way to change the stack size of this task by using
+    SET_LOOP_TASK_STACK_SIZE(size);
+    It will bypass the default stack size of 8KB and allow the user to define a new size.
+
+    It is recommend this value to be higher than 8KB, for instance 16KB.
+    This increasing may be necessary for the sketches that use deep recursion for instance.
+
+    In this example, you can verify it by changing or just commenting out SET_LOOP_TASK_STACK_SIZE();
+*/
+
+#define ARDUINO_TASK_STACK_SIZE     (16*1024)       // 16KB
```

Comment:
I will suggest to not add a separate define here and just do `SET_LOOP_TASK_STACK_SIZE(16*1024); // 16KB`

Reply:
done!